### DISCUSSION TITLE
[@types/newrelic]: Some properties are missing from types

### DISCUSSION THREAD

**Discussion by alexchoiweb** [DefinitelyTyped/DefinitelyTyped#69606] [Issues with a @types package]
I'm setting up new relic browser agent for a next.js app following this example:

[https://github.com/newrelic/newrelic-node-examples/blob/58f760e828c45d90391bda3f66764d4420ba4990/nextjs-app-router/app/layout.js](https://forum.newrelic.com/s/hubtopic/aAXPh0000001rgbOAA/void(0))

But getting type errors even after adding @types/newrelic, anyone know what I'm missing?

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/35116846/17e8d330-ca1c-4503-a208-8641c3672034)

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/35116846/22304089-314c-4bd2-b39b-0fa53f8ff31b)

![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/35116846/605fe7f5-7f95-41a9-891e-261e2048404d)

Discussed in https://github.com/newrelic/newrelic-node-examples/issues/230#issuecomment-2109979233

[a comment by an automated account is not shown]

**Reply by davidlewisbates1980** [reply]
Any news on this?

**Reply by doug-s-nava** [reply]
I am still seeing this issue as well. New Relic's documentation and sample applications reference attributes as shown by the OP above, but these types do not seem to be covered by the @types/newrelic. It seems like if these are parts of the New Relic system that they are directing users to use that they should be covered by type definitions. How should users expect to handle this?

**Reply by BrooksPatton** [reply]
I haven't been keeping up with New Relic for some time now. But in the past when some type information has changed, or is missing from here a PR can be created to update it.

**Reply by Chi8wah** [reply]
guys, may I ask how do you resolve the problem? It still occurs now. T T

**Reply by davidlewisbates1980** [reply]
We switched to [Sentry](https://sentry.io) :)
